script/build-brave-webtorrent.py

```diff
@@ -0,0 +1,51 @@
+import argparse
```

Comment:
we already have a `transpile_web_ui` action and it should be adapted if any changes are needed to handle this

Reply:
addressed by https://github.com/brave/brave-core/pull/353/commits/32c0fe277d781f756ab5581b213b01aef1986764